docs/mesos/ManualInstallCalicoDockerContainerizer.md

```diff
@@ -11,71 +11,66 @@
 This document provides the commands to download and run Calico
 for use with the Docker Containerizer in Mesos.
 
-## Prerequisites
+## Prerequisite: Docker
+Calico networks Docker tasks for Mesos with its libnetwork plugin. In order to
+run the libnetwork plugin, the Docker daemon on each agent must be configured
+with a cluster store.
 
-### Docker
-
-To use the multi-host native networking feature of Docker, the Docker daemon
-needs to be run with the cluster store parameter.  If using etcd as a cluster
-store, run the Docker daemon with the following additional parameter:
+If using etcd as a cluster store, for example, run the Docker daemon with the
+following additional parameter:
 
     --cluster-store=etcd://<ETCD HOST>:<PORT>
 
-Replacing `<ETCD HOST>:<PORT>` with the appropriate `IP/Hostname:Port`
+Replacing `<ETCD HOST>:<PORT>` with the appropriate `hostname:port`
 for your etcd cluster.
 
 ## Install and Run Calico
 It is very easy to install Calico to use with the
 Docker Containerizer.
 
-On each of your agents, you will need to download the `calicoctl` command-line tool:
-
-```
-curl -o /usr/bin/calicoctl -L https://github.com/projectcalico/calico-containers/releases/download/v0.18.0/calicoctl
-chmod a+x calicoctl 
-```
+1. On each Mesos Agents, download the `calicoctl` command-line tool:
 
-Now, you will need run the `calico/node` and
-`calico/node-libnetwork` containers.
+  ```
+  curl -o /usr/bin/calicoctl -L https://github.com/projectcalico/calico-containers/releases/download/v0.18.0/calicoctl
```

Comment:
Does this have to be v0.18.0?  If not, we should change this to point at latest (specifically http://www.projectcalico.org/builds/calicoctl).  Then when we cut a release it'll automatically pin to the version from that release.